ghost/external-media-inliner/lib/ExternalMediaInliner.js

```diff
@@ -39,14 +40,22 @@ class ExternalMediaInliner {
      * @param {string} requestURL - url of remote media
      * @returns {Promise<Object>}
      */
-    async #getRemoteMedia(requestURL) {
+    async getRemoteMedia(requestURL) {
         // @NOTE: this is the most expensive operation in the whole inlining process
         //        we should consider caching the results to improve performance
+
+        // Enforce https
+        requestURL = requestURL.replace(/^\/\//g, 'https://');
```

Comment:
What's going to happen if the file is not available over https? 🤔

Reply:
I've set this to http. http>https redirects are common, so I think this is fine

Reply:
It does seem fine, I'm just thinking of the edge case where https would be 404 for whatever bad reason 😂  Let's fix it when it breaks I guess. The web in 2023 should be all on https anyway!